### ISSUE TITLE
Dashed Polyline

### ISSUE THREAD

**Issue by fariz-siracli** [osmdroid/osmdroid#1187]
Hi. I want to draw dashed polyline. For this reason i write following code. (as @monsieurtanuki  suggested )
Polyline finishLine = new Polyline(map);
finishLine.setPoints(list2);
finishLine.getPaint().setColor(Color.RED);
finishLine.getPaint().setStyle(Paint.Style.STROKE);
finishLine.getPaint().setPathEffect(**new DashPathEffect(new float[]{10, 20}, 0)**);
But it draws ordinary line, not dashed. So what can do ?
Thanks for attention

**Comment by monsieurtanuki** [comment]
Seems to be a known [Android bug](https://code.google.com/p/android/issues/detail?id=29944) as they say in [SO](https://stackoverflow.com/questions/6103713/how-do-i-make-a-dotted-dashed-line-in-android).
Try to use a simple `canvas.drawLine` with the same `Paint`: it shouldn't work either.

We use `drawLine` for `Polyline`: you need to turn the hardware acceleration off, which should be something like that:
````java
mMapView.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
````

**Comment by fariz-siracli** [comment]
Do you mean to extend PolyLine 's draw method ?

**Comment by monsieurtanuki** [comment]
No, I mean just try to draw a line on a `Canvas` with that `Paint`, regardless of the map or of a polyline, just to check that the dash style won't appear because it's an Android bug and not because it's an osmdroid polyline. Allegedly.

**Comment by fariz-siracli** [comment]
Ok. i believe. But how about drawing dash line on Mapview ? Is'nt there  any other option ?

**Comment by monsieurtanuki** [comment]
@fariz-siracli From 6.1.0 you can create a polyline that uses `Path`: `public Polyline(final MapView pMapView, final boolean pUsePath)`.
Hopefully the `Path` version will correctly display the `Paint`. Can you please check that?
